### ISSUE TITLE
sdl_sound failed to build on 10.10.3/10.10.4

### ISSUE THREAD

**Issue by Keyaku** [Homebrew/homebrew#40872]
EDIT: Forgot to mention: this happens upon doing ```brew install sdl_sound --HEAD```

[Gist](https://gist.github.com/c4082857c8afc9600ccf).
```couldn't understand kern.osversion `14.3.0'``` (or 14.4.0 in OS X Yosemite 10.10.4).

It basically looks like a compatibility problem with the current version of Yosemite's kernel.

**Comment by DomT4** [comment]
I can't reproduce here. Are you still having the issue?

**Comment by DomT4** [comment]
Added bottles in https://github.com/Homebrew/homebrew/commit/19bbab6fd3ea81ca2772b6c1d4a212338c345515. Might make life a little easier here.

**Comment by Keyaku** [comment]
Agh, must've had a terrible night when I wrote this. I'm really sorry.
I forgot to mention that it fails to build with the HEAD flag. It still doesn't build.

**Comment by DomT4** [comment]
Ah :smile_cat:. No failure here still:

```
==> Summary
🍺  /usr/local/Cellar/sdl_sound/HEAD: 10 files, 664K, built in 79 seconds
```

Can you `brew gist-logs sdl_sound` after attempting a `--HEAD` build? The existing gist-logs seems to point to a stable build. (Although it's unclear why that failed for you. The error was kind of ambiguous).

**Comment by Keyaku** [comment]
The gist-logs are pretty similar to what I've published before.
I've decided to take a look at the the Gist. Since it breaks at `make`, I know where to look.  
My conclusion is that I've been gullible to believe it was a kernel incompatibility; now that I have somewhat proper time to look at it, the problem seems to be nothing related to this, but rather at:
`./physfsrwops.h:42:1: error: unknown type name '__EXPORT__'`

It's that line and a few others that's making the problem. Somehow, it doesn't know what \__EXPORT__ is, and I can't seem to find what I can do to fix it. Maybe it's my build of physfs that's causing the problem?
